### PR TITLE
feat(components/PhoneNumberAuth): Creating forms for authorization us…

### PR DESCRIPTION
**Pull request by glemmaPaul**
…ing otp

### CODE REVIEW

File: src/components/PhoneNumberAuth/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  )
+}
+
+export default function(props) {
+    const [phoneNumber, setPhoneNumber] = useState('')
+    const [confirmationResult, setConfirmationResult] = useState(null)
+    const [code, setCode] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const [user, setUser] = useState(null)
+    const [loading, setLoading] = useState(false)
```
Comment (cdkirkwood) [inline src/components/PhoneNumberAuth/index.js:19]: Maybe replace with suspense?
https://reactjs.org/docs/code-splitting.html#suspense

File: src/components/PhoneNumberAuth/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                    type="tel"
+                    value={phoneNumber}
+                    placeholder="Phone Number"
+                    className="form-input fs16 bg-white c-gold mb-"
+                    onChange={(e) => setPhoneNumber(e.target.value)}
+                    /> <span hidden={errorMessage === ''} style={{ "color": "red" }}>{errorMessage}</span>
+                <button type="submit" className="btn btn--primary" disabled={loading} id="sign-in-button">{ loading ? 'Sending..' : 'Send Text' }</button>
+            </form>
+            :
+            <form>
```
Comment (cdkirkwood) [inline src/components/PhoneNumberAuth/index.js:91]: form doesn't have onSubmit. Probably want to set this to the button's onClick

File: src/components/PhoneNumberAuth/index.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    function onResendPhoneNumber(e) {
+      setCode(null)
+      setConfirmationResult(null)
+    }
+
+    return (
+      <div>
+        {
+          (confirmationResult === null) ?
+            <form onSubmit={onSubmitPhone}>
```
Comment (cdkirkwood) [inline src/components/PhoneNumberAuth/index.js]: These two form elements could be made into one reusable component.

File: src/components/PhoneNumberAuth/index.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react'
+import firebase, { firebaseErrorResponse } from '../../firebase'
+
+
+function PhoneNumberSuccess(props) {
```
Comment (cdkirkwood) [inline src/components/PhoneNumberAuth/index.js:5]: Might be better to have this in its own file, if it needs to be its own component at all.

File: src/components/PhoneNumberAuth/index.js (hunk 1 above)
Comment (glemmaPaul) [reply src/components/PhoneNumberAuth/index.js:19]: Yeah Let me check how that can work out, good point!

File: src/components/PhoneNumberAuth/index.js (hunk 2 above)
Comment (glemmaPaul) [reply src/components/PhoneNumberAuth/index.js:91]: Yeah I messed that one up. It does work onSubmit, but it isnt that reliable it seems like

File: src/components/PhoneNumberAuth/index.js (hunk 1 above)
Comment (glemmaPaul) [reply src/components/PhoneNumberAuth/index.js:19]: Will probably make an other PR out of this so we can get this merged

File: src/components/PhoneNumberAuth/index.js (hunk 4 above)
Comment (glemmaPaul) [reply src/components/PhoneNumberAuth/index.js:5]: it's a little silly :D

File: src/components/PhoneNumberAuth/index.js (hunk 2 above)
Comment (glemmaPaul) [reply src/components/PhoneNumberAuth/index.js:91]: Updated

**Pull request merged**